http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -449,15 +494,52 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         // Enabled services configs
         raw"cbas.enabled=${config.coaAppConfig.coaServices.contains(Cbas)}",
         raw"cbasUI.enabled=${config.coaAppConfig.coaServices.contains(CbasUI)}",
-        raw"wds.enabled=${config.coaAppConfig.coaServices.contains(Wds)}",
+        raw"wds.enabled=${config.coaAppConfig.coaServices.contains(Wds)}", // TODO remove after WDS chart migration
```

Comment:
As far as migration:
- first we'll deploy these changes adding support for a WDS only app
- next we'll deploy a change shifting autocreation to autocreate the WDS only app. This be in rawls or TerraUI, depending on what mechanism we're using for autodeploy
- *How does CBAS find the WDS URL? We'll need to make sure CBAS is in sync for the migration
- finally, we can remove the logic for WDS in the cromwell App in leo, and remove the helmchart from cromwhelm

Reply:
claimed!